### PR TITLE
add OS X to travis configs

### PR DESCRIPTION
**Pull request by reaperhulk**

### CODE REVIEW

File: .travis/install.sh [hunk 1]
```diff
@@ -3,13 +3,36 @@
 set -e
 set -x
 
-if [[ "${TOXENV}" == "pypy" ]]; then
+install_pypy () {
```
Comment (alex) [inline .travis/install.sh]: This function does not appear to install PyPy?

File: .travis/install.sh [hunk 2]
```diff
@@ -3,13 +3,36 @@
...
     git clone https://github.com/yyuu/pyenv.git ~/.pyenv
     PYENV_ROOT="$HOME/.pyenv"
     PATH="$PYENV_ROOT/bin:$PATH"
     eval "$(pyenv init -)"
-    pyenv install pypy-2.6.0
-    pyenv global pypy-2.6.0
+}
+
+if [[ "$(uname -s)" == 'Darwin' ]]; then
+    install_pypy
```
Comment (alex) [inline .travis/install.sh]: I assume we should only install pypy if we are using it?

File: .travis/run.sh [hunk 3]
```diff
@@ -3,11 +3,20 @@
 set -e
 set -x
 
-if [[ "${TOXENV}" == "pypy" ]]; then
+init_pypy () {
```
Comment (alex) [inline .travis/run.sh]: This does not initialitize pypy

Comment (reaperhulk) [inline .travis/run.sh]: pypy, pyenv, what's the difference! (Oops)

**Pull request merged**